django_mock_queries/query.py

```diff
@@ -30,8 +30,9 @@ def MockSet(*initial_items, **kwargs):
         'prefetch_related',
         'select_for_update'
     ])
-    mock_set.cls = clone.cls if clone else kwargs.get('cls', empty_func)
+    mock_set.cls = clone.cls if clone else kwargs.get('cls', MockModel)
```

Comment:
Shall we remove `.cls` now that we have `. model` that covers everything? It has a `_meta` with all the fields required for `create` and `get_or_create` etc.